src/Mvc/Mvc.ViewFeatures/src/HtmlHelperRazorComponentExtensions.cs

```diff
@@ -1,22 +1,21 @@
 ﻿// Copyright (c) .NET Foundation. All rights reserved.
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
-using System.Collections.Generic;
-using System.IO;
-using System.Text.Encodings.Web;
+using System;
 using System.Threading.Tasks;
 using Microsoft.AspNetCore.Components;
-using Microsoft.AspNetCore.Components.Rendering;
+using Microsoft.AspNetCore.Components.Server;
 using Microsoft.AspNetCore.Html;
 using Microsoft.AspNetCore.Mvc.Rendering;
+using Microsoft.AspNetCore.Mvc.ViewFeatures.RazorComponents;
 using Microsoft.Extensions.DependencyInjection;
 
 namespace Microsoft.AspNetCore.Mvc.ViewFeatures
 {
     /// <summary>
     /// Extensions for rendering components.
     /// </summary>
-    public static class HtmlHelperComponentExtensions
+    public static class HtmlHelperRazorComponentExtensions
```

Comment:
ehhh I think it was better before.

Reply:
I want to distinguish between ViewComponents and MAC Components, so using the razor prefix felt better. I don't think it matters this much here as you won't use this class by name. If you feel strongly about it i can revert the name change.

Reply:
We can leave it for now.